src/modules/commander/Commander.cpp

```diff
@@ -4042,6 +4046,15 @@ void Commander::esc_status_check()
 	_last_esc_status_updated = esc_status.timestamp;
 }
 
+void Commander::send_parachute_command()
+{
+	send_vehicle_command(vehicle_command_s::VEHICLE_CMD_DO_PARACHUTE,
```

Comment:
Are you sure these commands get sent out over MAVLink that way?
I was assuming that `send_vehicle_command()` is only for internal commands but I might be mistaken. Isn't that also why it would always use the system and component ID of itself?

Reply:
It gets sent in my testing.

Reply:
Tested and it gets sent, I also checked the code in the mavlink module that sends it out 👍